packagelist/ubuntu_23.04_headless.txt

```diff
@@ -78,3 +78,5 @@ libubootenv-tool
 lsof
 meson
 nfs-common
+sqlite3
+xxd
```

Comment:
Make sure any tools you're adding for Debian also get added to the Ubuntu lists so these images are mostly consistent.